src/gui/SubWindow.cpp

```diff
@@ -39,29 +45,77 @@ SubWindow::SubWindow(QWidget *parent, Qt::WindowFlags windowFlags)
 	m_trackedNormalGeom = normalGeometry();
 }
 
+
+
+
+void SubWindow::paintEvent(QPaintEvent *)
+{
+	QPainter painter(this);
+	QRectF rect(0,0,width()-1,20);
```

Comment:
May I ask why it's using `width()-1`? I noticed there is a weird black 1px line on the right side of the title bar before I changed it to `width()`